KDT0_YoonSeokMin 대학교 홈페이지 클론 코딩

dbstjrals opened the pull request:
## 대학교 유세인트 클론코딩

### 실제 사이트

https://saint.ssu.ac.kr/irj/portal

### 배포 사이트

https://gorgeous-alfajores-250a00.netlify.app/

---

<br/>
float 속성을 이용하여 요소들을 정렬하고
<br/>
line-height나 text-align 등을 이용해서 정렬을 하던 기존 방식에서
<br/>
flex를 이용하여 변경했습니다  
<br/><br/>
평소 ul li 태그를 사용해 본 적이 많이 없어서
<br/>
공지사항 부분은 flex 없이 ul li 태그로 구현했습니다

dbstjrals committed:
Feat: 클론 코딩 및 배포

bu-mang commented:
css파일을 font, general, main 나눠서 관리하는 것 참고가 되네요. 매우 재현율도 높고 멋집니다!

TaePoong719 commented on styles/main.css:
```diff
@@ -0,0 +1,492 @@
+.login_right_box {
+  position: relative;
+  max-width: 757px;
+  top: 24%;
```
개인 로컬환경에서 login_right_box박스가 아래로 나가더라고요, display flex를 이용하여 중앙을 잡는 것이 추후 반응형 작업을 하나 줄여 주는데 도움을 줄 것 같습니다

TaePoong719 reviewed (commented):
다양한 태그들을 용도에 맞게 잘 사용하십니다. 아프신 와중에 고생 많으셨습니다. 독특한 사이트를 하셨네요 ㅎ.ㅎ

JIYEONGYANGdev commented on styles/main.css:
```diff
@@ -0,0 +1,492 @@
+  transition: all 0.5s ease;
+}
+.login_right_box {
+  position: relative;
```
display: relative 를 사용하셨네요. 화면 상 가운데에 보이기 위해 지정해준 것 같은데요.

예시 웹사이트는 상단에 헤더나 내비게이션이 없긴 하지만, 
최상위 요소에서 상단일정 margin을 주는 식으로 처리해도 괜찮을 것 같네요.

![숭실대학교 유세인트(u-SAINT) - Chrome 2023-08-01 오후 6_20_15](https://github.com/KDT1-FE/Y_FE_HTML_CSS/assets/82949221/5e454d8c-d286-4308-84ef-c460b58bc3f4)

display: relative를 중복적으로 사용하다가 꼬일 수가 있어서 주의하면 좋을 것 같습니다. 부모/자식 요소간 display: absolute - relative 활용하는 것을 좀 더 찾아보면 좋을 것 같아요.

JIYEONGYANGdev reviewed (commented):
추후에 반응형 개발 시에 display: flex, relative를 더 제대로 활용해볼 수 있을 거예요!

작업에 대해 Readme를 좀 더 자세히 작성하는 걸 추천드립니다:)
수고하셨습니다~

moana16 commented:
익숙한 페이지여서 재밌네요! 근데 코드 가독성을 위해 시멘틱태그를 사용해보시는 것도 좋을 것같아요! 저도 맨날 div 태그만 사용하다가 이번에 사용해보았는데 훨씬 가독성이 높아지고 구조가 한눈에 보여서 좋더라구요

JSH99 commented on styles/main.css:
```diff
@@ -0,0 +1,492 @@
+  white-space: nowrap;
+  overflow: hidden;
+}
+.tab_box01 .li02 .notice_list ul li .title:before {
```
:before와 ::before의 차이점이 궁금해서 찾아보니 같은 동작을 하지만 버전 차이라고 하네요! ::before 문법을 더 권장한다고 하니 관련 레퍼런스 찾아 보시면 좋을 것 같아요😁

JSH99 reviewed (commented):
image 및 css 디렉토리 정리가 깔끔해서 보기 좋은 것 같아요! 주석으로도 정리해놓으셔서 코드 가독성이 좋았습니다👏 몸 안 좋으신데 과제하시느라 고생하셨어요🥺

dbstjrals committed:
Style: 시멘틱 태그 사용 및 요소 위치 지정 방법 변경

login_left와 login_right 클래스명을 가진 div 태그를 section 태그로 변경

content의 위치를 display: relative와 top 속성을 사용해서 지정했었는데
해당 속성들 대신 margin-top 속성을 통해 화면상 가운데에 위치 시킴
다만 기존 div에서 margin을 줄 경우 background image나 color에 영향이 생겨
content만을 담고 있는 하위 div에서 margin-top 속성을 추가하였음

dbstjrals committed:
Refactor: 1200px 이하 반응형 웹 디자인 추가

dbstjrals committed:
Fix: left margin 속성값 단위를  %에서 px로 변경
